documentation/catalog-content.md

```diff
@@ -95,6 +95,13 @@ Feel free to add suggestions for new slugs to the lists as part of your PR! Be s
 - 'paths/front-end-engineer-career-path'
 ```
 
+### JavaScript:D3
+
+```
+- 'learn-d3'
+- 'paths/data-science'
+```
+
```

Comment:
I'll let you know about this later because we don't have the slugs available in the CatalogContent.md file. The slugs you have added need to be confirmed by the internal team.

Reply:
This is fine, we can send it to the second review.